docs/concepts/resource-manager.md

```diff
@@ -1185,3 +1185,25 @@ When this webhook is activated, it reacts for most of the custom resources in th
 It also reacts for the `druid.gardener.cloud/v1alpha1.Etcd` resources.
 
 The webhook validates the resources specifications for `CREATE` and `UPDATE` requests.
+
+### Authorization Webhooks
+
+#### `node-agent-authorizer` webhook
+
+`gardener-resource-manager` serves an authorization webhook for shoot `kube-apiserver` in order to grant access to `gardener-node-agent`.
+It works similar to [`SeedAuthorizer`](https://github.com/gardener/gardener/blob/master/docs/deployment/gardenlet_api_access.md), but decisions are much simpler so it does not need to implement a decision graph.
+In many cases, the objects `gardener-node-agent` is allowed to access, depend on the `Node` it is running on.
+
+`Machine.Name` is used as unique username of `gardener-node-agent`. It follows this pattern `gardener.cloud:node-agent:machine:<machine-name>`.
+The reference to a `Node` is determined from `node` label of the `Machine`.
+All users are assigned to `gardener.cloud:node-agents` group. Extensions could use this group to grant permissions to all `gardener-node-agents`.
```

Comment:
I was thinking about [SecretRefs in OSC](https://github.com/gardener/gardener/blob/b2e0cd2f1634509f7284dad6bbfb8d401cdc11cf/pkg/apis/extensions/v1alpha1/types_operatingsystemconfig.go#L150) which could be added by extensions.
However, this does not make any sense 😅 
The secret from the SecretRef are converted into `inline` files because they are in shoot namespace of the seed, where GNA does not have access to. 

I removed the sentence. For now, the group will be required for the migration case.